README.md

```diff
@@ -1 +1,162 @@
+
+# 📖 Morse Decode <a name="about-project"></a>
+
+**Morse Decode** is a ruby written program to decode any morse message. It takes up any morse coded letters, word or message and spits out the decoded letter, word and message.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>None</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Ruby</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>None</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **DRY method**
+- **Methods for decoding morse coded message**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+https://github.com/firstpj/Morse-code.git
+
+### Install
+
+
+```sh
+  cd Morse-Code
+  bundle install
+```
+
+
+### Usage
+
+To run the project, execute the following steps:
+
+```sh
+  ruby decode.rb
+```
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Mohammad Ali Reza**
+
+- GitHub: [@githubhandle](https://github.com/AliRezaBmeDu)
+- Twitter: [@twitterhandle](https://twitter.com/share_insider)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mohammad-ali-reza-25686428b/)
+
+👤 **Collins Akorda**
+
+- GitHub: [@githubhandle](https://github.com/firstpj)
+- Twitter: [@twitterhandle](https://twitter.com/AkordaC)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/collins-akorda-bb46b2232/)
+
+👤 **Fatuma Hussein**
+
+- GitHub: [@githubhandle](https://github.com/fatmahussein)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/fatuma-hussein-48149917b)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Make it viable to a web app**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/AliRezaBmeDu/Morse-code/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it a ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank all the documents provided by microverse.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] 🌐 IMPORTANT: Please make sure all links within the README file are functional. It's crucial that all links work as intended to provide a seamless experience for users and contributors. Broken links can lead to confusion and frustration. Let's ensure everything is accessible! 💪